src/main/java/com/iexec/sms/admin/AdminService.java

```diff
@@ -149,4 +143,72 @@ boolean restoreDatabaseFromBackupFile(String storageLocation, String backupFileN
         }
         return false;
     }
+
+    /**
+     * Delete a backup of the H2 database from a location
+     *
+     * @param storageLocation The location of the backup file.
+     * @param backupFileName  The name of the backup file.
+     * @return {@code true} if the deletion was successful; {@code false} if any error occurs.
+     */
+    public boolean deleteBackupFileFromStorage(String storageLocation, String backupFileName) {
+        try {
+            // Ensure that storageLocation and backupFileName are not blanks
+            boolean validation = commonsParametersValidation(storageLocation, backupFileName);
+            if (!validation) {
+                return false;
+            }
+            // Ensure that storageLocation ends with a slash
+            storageLocation = normalizePathWithSeparator(storageLocation);
```

Comment:
This part may be unnecessary with `storageLocation + File.separator + backupFileName`, then `getCanonicalPath()`.
What do you think ?

Reply:
you're right